recipe_sha: 818b19264e031ab0aafc69a65856602ab4de67b68aed3fe7e3117d898e0765f6  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
      libffi:          3.4.2-h0d85af4_5           conda-forge
      libsqlite:       3.42.0-h58db7d2_0          conda-forge
      libzlib:         1.2.13-hfd90126_4          conda-forge
      ncurses:         6.3-h96cf925_1             conda-forge
      openssl:         3.1.1-h8a1eda9_1           conda-forge
      pip:             23.1.2-pyhd8ed1ab_0        conda-forge
      python:          3.10.11-he7542f4_0_cpython conda-forge
      readline:        8.2-h9e318b2_1             conda-forge
      setuptools:      67.7.2-pyhd8ed1ab_0        conda-forge
      tk:              8.6.12-h5dbffcc_0          conda-forge
      tzdata:          2023c-h71feb2d_0           conda-forge
      wheel:           0.40.0-pyhd8ed1ab_0        conda-forge
      xz:              5.2.6-h775f41a_0           conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  Reloading output folder: /opt/mambaforge/envs/bioconda/conda-bld
  Reloading output folder: /opt/mambaforge/envs/bioconda/conda-bld
  Mamba failed to solve:
   - bedtools
   - numpy
   - scipy
   - samtools
   - matplotlib 3.3.*
   - python_abi 3.10.* *_cp310
   - pysam
   - python >=3.10,<3.11.0a0

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package matplotlib-3.3.1-0 requires matplotlib-base >=3.3.1,<3.3.2.0a0, but none of the providers can be installed
  -

  Leaving build/test directories:
    Work:
   /opt/mambaforge/envs/bioconda/conda-bld/work
    Test:
   /opt/mambaforge/envs/bioconda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/mambaforge/envs/bioconda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla
    Build:
  source activate  /opt/mambaforge/envs/bioconda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/boa/core/solver.py", line 230, in solve_for_action
      t = self.solve(specs)
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/boa/core/solver.py", line 220, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - bedtools
   - numpy
   - scipy
   - samtools
   - matplotlib 3.3.*
   - python_abi 3.10.* *_cp310
   - pysam
   - python >=3.10,<3.11.0a0

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package matplotlib-3.3.1-0 requires matplotlib-base >=3.3.1,<3.3.2.0a0, but none of the providers can be installed
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/mambaforge/envs/bioconda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/conda_build/build.py", line 2115, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/conda_build/build.py", line 1993, in create_build_envs
      raise e
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/conda_build/build.py", line 1972, in create_build_envs
      environ.get_install_actions(m.config.test_prefix,
    File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform osx-64: {MatchSpec("matplotlib==3.3.1=0"), MatchSpec("matplotlib-base[version='>=3.3.1,<3.3.2.0a0']")}
# Last 100 lines of the build log.
